pkgs/liquify/CHANGELOG.md

```
@@ -1,3 +1,11 @@
+## 1.6.0
+
+### Browser Support
+- **Make the package usable from browser builds** - Split the filesystem exports so `webdev`/browser entrypoints no longer pull in `dart:io`, allowing the package to be used more freely in the browser and in the playground.
+
+### Bug Fixes
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)